lib/archethic/contracts/interpreter/function_interpreter.ex

```diff
@@ -0,0 +1,127 @@
+defmodule Archethic.Contracts.Interpreter.FunctionInterpreter do
+  @moduledoc false
+  alias Archethic.Contracts.Interpreter.Scope
+  alias Archethic.Contracts.Interpreter.CommonInterpreter
+  require Logger
+
+  @doc """
+  Parse the given node and return the function name it's args as strings and the AST block.
+  """
+  def parse(_, _ \\ [])
```

Comment:
I think this default value is not necessary since the function_key are always passed as parameter